src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -252,40 +276,49 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
         return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
-    private static _renderImageCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element {
-        const textToRender: string | undefined = deployment.image;
-        const itemToRender = BaseKubeTable.renderColumn(textToRender || "", BaseKubeTable.defaultColumnRenderer, colDataClassName);
+    private _renderImageCell=(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element=> {
+        const textToRender: string = deployment.imageDisplayText;
+        const imageId: string = deployment.imageId;
+        // HardCoding hasImageDetails true for the time being, Should change it once we integrate with ImageService
+        //const hasImageDetails: boolean = this._hasImageDetails && this._hasImageDetails.hasOwnProperty(firstImageName) ? this._hasImageDetails[firstImageName] : false;
+        const hasImageDetails = true;
+        const itemToRender = hasImageDetails ?
+            (<Link
+                className="fontSizeM text-ellipsis bolt-table-link bolt-table-inline-link"
+                onClick={() => this._onImageClick(this.props.imageService, imageId)}>
+                {textToRender || ""}
+            </Link>) :
+            BaseKubeTable.renderColumn(textToRender || "", BaseKubeTable.defaultColumnRenderer, colDataClassName);
         return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
     private static _renderAgeCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element {
-        const textToRender: string | undefined = deployment.image;
         const itemToRender = (<Ago date={new Date(deployment.creationTimeStamp)} />);
         return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
     private static _getHeadingContent(deployment: V1Deployment): JSX.Element {
         return (
-                <span>
-                   {deployment.metadata.name}
-                    <div className="deployment-heading-labels">
-                        {<LabelGroup labelProps={Utils.getUILabelModelArray(deployment.metadata.labels)}
-                            wrappingBehavior={WrappingBehavior.oneLine}
-                            fadeOutOverflow={true}>
-                        </LabelGroup>}
-                    </div>
-                </span>
+            <span>
```

Comment:
we may need to change this

Reply:
Will revisit thisonce your PR is checked in